src/main/kotlin/no/nav/omsorgspengerutbetaling/soknad/Søknad.kt

```diff
@@ -8,20 +8,18 @@ import java.util.*
 data class Søknad(
     val søknadId: SøknadId = SøknadId(UUID.randomUUID().toString()),
     val språk: Språk,
-    val harDekketTiFørsteDagerSelv: Boolean? = null, // TODO: 08/04/2021 Fjern nullable etter prodsetting.
+    val harDekketTiFørsteDagerSelv: Boolean,
```

Comment:
Kan vurdere å sette denne til default null og validere at den faktisk blir satt. For å forhindre default false ved feil